packages/core-components/src/components/articles/ArticleList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Article, { ArticleType } from './Article';
+
+type ArticleListProps = {
+  articles: Array<ArticleType>;
+};
+
+const ArticleList: React.FC<ArticleListProps> = ({
+  articles
+}) => {
+  return (
+    <div className='space-y-2 columns-1 md:columns-2'>
+      {
+        articles.map((article) => {
+          return (
+            <Article
+              article={article}
+            />
+          );
+          })
+      }
+    </div>
```

Comment:
```suggestion
      {
        articles.map((article) =>  (
            <Article
              article={article}
            />
          ))
      }
    </div>
```
small, can resolve in later PR